assume/common/outputs.py

```diff
@@ -40,18 +40,18 @@ def __init__(
         start: datetime,
         end: datetime,
         db_engine=None,
-        export_csv_path: str = "",
-        save_frequency_hours: int = 24,
+        export_csv_path: str = None,
+        save_frequency_hours: int = None,
         learning_mode: bool = False,
     ):
         super().__init__()
 
         # store needed date
         self.simulation_id = simulation_id
-        self.save_frequency_hours = save_frequency_hours
+        self.save_frequency_hours = save_frequency_hours or (end - start).days * 24
 
         # make directory if not already present
-        self.export_csv_path = export_csv_path
+        self.export_csv_path = export_csv_path or 0
         if self.export_csv_path:
             self.p = Path(self.export_csv_path, simulation_id)
             shutil.rmtree(self.p, ignore_errors=True)
```

Comment:
here we are removing the folder and recreating it properly

Reply:
as I said before, on my system defining export_csv_path as "" makes a no name folder. This fixes it

Reply:
so the problem was actually because of the mkdir in the main examples.py file and not here. I have now removed it from the main file and it seems to work fine